WalletWasabi.Daemon/Config.cs

```diff
@@ -41,6 +41,7 @@ public Config(PersistentConfig persistentConfig, string[] cliArgs)
 			{ nameof(JsonRpcUser), GetStringValue("JsonRpcUser", PersistentConfig.JsonRpcUser, cliArgs) },
 			{ nameof(JsonRpcPassword), GetStringValue("JsonRpcPassword", PersistentConfig.JsonRpcPassword, cliArgs) },
 			{ nameof(JsonRpcServerPrefixes), GetStringArrayValue("JsonRpcServerPrefixes", PersistentConfig.JsonRpcServerPrefixes, cliArgs) },
+			{ nameof(OnionEnabled), GetBoolValue("OnionEnabled", value: false, cliArgs) },
```

Comment:
Maybe `OnionServerEnabled`, to not confuse with the client side `usetor`?

Reply:
Done.